app.js

```diff
@@ -9,4 +9,121 @@ hamburger.addEventListener('click', () => {
 document.querySelectorAll('.nav-link').forEach((n) => n.addEventListener('click', () => {
   hamburger.classList.remove('active');
   navMenu.classList.remove('active');
-}));
\ No newline at end of file
+}));
+
+// JS for Form Validation
+const name = document.getElementById('fullname');
+const email = document.getElementById('email-address');
+const form = document.getElementById('contact');
+const errorElement = document.getElementById('small');
+
+form.addEventListener('submit', (e) => {
+  if (name.value === '' || name.value == null) {
+    e.preventDefault();
+    errorElement.innerText = 'Please enter your name';
+  } else if (email.value === '' || email.value == null) {
+    e.preventDefault();
+    errorElement.innerText = 'Please enter email';
+  } else if (email.value === email.value.toUpperCase()) {
+    e.preventDefault();
+    errorElement.innerText = 'Please use Lowercase';
+  }
+});
```

Comment:
You did well with your project, however your validation seems to not be working properly. when a user inputs an email address with a mixture of uppercase and lowercase letters, the form seems to submit, however when a user uses only uppercase letters, it doesn't submit. Please ,according to the project requirement, the form is supposed to check for validation even if it has just one uppercase letter 

Project requirement : 
![image](https://github.com/anyars-encarta/anyars-encarta.github.io/assets/20537712/0e219622-ee6f-4f5f-b499-90712b0cce55)

Your validation :

https://github.com/anyars-encarta/anyars-encarta.github.io/assets/20537712/45758bb3-f5b5-4c87-bd55-027b29ba2a50